oneview_redfish_toolkit/authentication.py

```diff
@@ -42,7 +44,12 @@ def init_map_tokens():
 
 
 def _set_new_token(redfish_token, tokens_ov_by_ip):
-    globals()['map_tokens'][redfish_token] = tokens_ov_by_ip
+    lock = Lock()
```

Comment:
Why do not use `with lock` instead?
The code would be:
```python
with lock:
    globals()['map_tokens'][redfish_token] = tokens_ov_by_ip
```
This statement, implicitly, will apply `lock.acquire()` and `lock.release()` as is said [here](https://docs.python.org/3/library/threading.html#using-locks-conditions-and-semaphores-in-the-with-statement).

Reply:
You are right, I want to implement it in a more explicit way, but we sure can change it.

Reply:
what do you think?
I want you to feel at ease. I prefer another way because of simplicity.
But, the code has not problem for me. We already can merge it if all agree, to. =)